src/components/Dashboard/Sidebar.tsx

```diff
@@ -0,0 +1,164 @@
+    },
+  ];
+  const datadown = [{
+    icon: FaRegSun,
+    name: 'Settings',
+  },
+  {
+    icon: FaRegUser,
+    name: 'Account',
+  },
+  {
+    icon: FaSignOutAlt,
+    name: 'Logout',
+  },
+  {
+    icon: FaMoon,
+    name: 'switch mode',
+  },
+
+  ];
+  return (
+
+    <div>
+      {!isOpen && (
+        <motion.div
+          onClick={() => setOpen(true)}
+          className="absolute w-fit h-fit z-50 left-0 top-19 cursor-pointer "
+        >
+          { /* for close and open botton */}
+          <MdOutlineApps size={30} className="fill-cyan-500" />
+        </motion.div>
+      )}
+
+      <motion.div
+        variants={sidebarAnimation}
+        animate={isOpen ? 'open' : 'closed'}
+        className="flex flex-low h-[100vh] px-5 py-2 overflow-hidden md:relative fixed"
+      >
+
+        {/* for controlling button */}
+        <motion.div
+          onClick={() => setOpen(false)}
+          className="absolute w-fit h-fit z-50 right-0 top-2 cursor-pointer border-slate-300"
+        >
+          { /* for close */}
+          <FaOutdent size={20} className="fill-cyan-500" />
+        </motion.div>
+
+        {/* menu */}
+        <div className="flex flex-col h-full ">
+          {/* first */}
+          <ul className="opacity-60 text-[0.9rem] py-5 flex flex-col gap-2
+           font-medium overflow-x-hidden scrollbar-thin scrollbar-track-white scrollbar-thumb-slate-100 h-[100%] md:h-[68%] "
+          >
+            <li>
+              <NavLink to="/dashboard" className="p-2.5 flex items-center md:cursor-pointer  flex flex-row gap-3  dark:text-white dark:hover:text-primary hover:text-primary">
+                <FaTh size={23} className="min-w-max" />
+                Dasboard
+              </NavLink>
+            </li>
+
+            {/* with submenu */}
+
+            {isOpen && (
+              <div className="border-y py-5 border-slate-300">
+                <small className=" pl-3 text-slate-500 inline-block mb-2">
+                  Features
+                </small>
+                {
+                subMenuList?.map((menu) => (
+                  <div key={menu.name} className=" flex flex-col gap-1">
+                    <SubMenu name={menu.name} icon={menu.icon} menus={menu.menus} />
+
+                  </div>
+
+                ))
+                }
+
+              </div>
+            )}
+
+            <li>
+              <NavLink to="/Notification" className="p-2.5 flex rounded-md items-center md:cursor-pointer  duration-300 font-medium flex-row gap-3 dark:text-white dark:hover:text-primary hover:text-primary">
+                <FaBullhorn size={23} className="min-w-max" />
+                Notification
+              </NavLink>
+            </li>
+            <li className="hover:text-secondary duration-300">
+              <NavLink to="/messages" className="p-2.5 flex rounded-md items-center md:cursor-pointer font-medium flex-row gap-3 dark:text-white dark:hover:text-primary hover:text-primary">
```

Comment:
These are all classnames using the same styles. Can you store them in one variable above and just pass it around. It will help read the code easier.